ts/hash_to_field.ts

```diff
@@ -0,0 +1,209 @@
+    },
+    {
+        msg: "abc",
+        outLen: 128,
+        expected:
+            "0xfe994ec51bdaa821598047b3121c149b364b178606d5e72bfbb713933acc29c186f316baecf7ea22212f2496ef3f785a27e84a40d8b299cec56032763eceeff4c61bd1fe65ed81decafff4a31d0198619c0aa0c6c51fca15520789925e813dcfd318b542f8799441271f4db9ee3b8092a7a2e8d5b75b73e28fb1ab6b4573c192"
+    },
+    {
+        msg: "abcdef0123456789",
+        outLen: 128,
+        expected:
+            "0xc9ec7941811b1e19ce98e21db28d22259354d4d0643e301175e2f474e030d32694e9dd5520dde93f3600d8edad94e5c364903088a7228cc9eff685d7eaac50d5a5a8229d083b51de4ccc3733917f4b9535a819b445814890b7029b5de805bf62b33a4dc7e24acdf2c924e9fe50d55a6b832c8c84c7f82474b34e48c6d43867be"
+    },
+    {
+        msg:
+            "q128_qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqq",
+        outLen: 128,
+        expected:
+            "0x48e256ddba722053ba462b2b93351fc966026e6d6db493189798181c5f3feea377b5a6f1d8368d7453faef715f9aecb078cd402cbd548c0e179c4ed1e4c7e5b048e0a39d31817b5b24f50db58bb3720fe96ba53db947842120a068816ac05c159bb5266c63658b4f000cbf87b1209a225def8ef1dca917bcda79a1e42acd8069"
+    },
+    {
+        msg:
+            "a512_aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
+        outLen: 128,
+        expected:
+            "0x396962db47f749ec3b5042ce2452b619607f27fd3939ece2746a7614fb83a1d097f554df3927b084e55de92c7871430d6b95c2a13896d8a33bc48587b1f66d21b128a1a8240d5b0c26dfe795a1a842a0807bb148b77c2ef82ed4b6c9f7fcb732e7f94466c8b51e52bf378fba044a31f5cb44583a892f5969dcd73b3fa128816e"
+    }
+];
+import { assert } from "chai";
+function test_expand_msg() {
+    for (let i = 0; i < vectors.length; i++) {
+        const v = vectors[i];
+        const msg = Uint8Array.from(Buffer.from(v.msg, "utf8"));
+        const outLen = v.outLen;
+        const out = expandMsg(DST, msg, outLen);
+        assert.equal(hexlify(out), v.expected);
+    }
+}
+function test_hash_to_point() {
+    const expected0 =
+        "0x09b6a2dec1f1b0747c73332e5147ecacde20767f28a9b68261713bed9a1d2432";
+    const expected1 =
+        "0x0cb70ff0b1bdb5d30006bd0cc03dc2c071dcff0daea886c9793f304c695c1bc6";
+    const dst = "xxx";
+    const msg = "0x616263";
+    mcl.setDomain(dst);
+    const p = mcl.hashToPoint(msg);
+    const result = mcl.g1ToHex(p);
+    assert.equal(result[0], expected0);
+    assert.equal(result[1], expected1);
+}
+
+import * as mcl from "./mcl";
+async function test() {
```

Comment:
Can we organize these test related functions to the `test/`?

Reply:
those tests test tests :). 
I've left them for reviewers or auditors to prove that we are in line with the standart. These test don't need be run repeatedly.

Reply:
I see. I'll later organize them to #202.